src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.Utf8JsonStreamReader.cs

```diff
@@ -225,6 +227,29 @@ internal static PackageSpec GetPackageSpec(ref Utf8JsonStreamReader jsonReader,
                             }
                         }
                     }
+                    else if (jsonReader.ValueTextEquals(UsingMicrosoftNETSdk))
+                    {
+                        if (packageSpec.RestoreMetadata is null)
+                        {
+                            packageSpec.RestoreMetadata = new ProjectRestoreMetadata();
+                        }
+
+                        packageSpec.RestoreMetadata.UsingMicrosoftNETSdk = jsonReader.ReadNextTokenAsBoolOrTrue();
```

Comment:
As commented elsewhere, what's the error experience when other boolean properties have values, but can't be parsed as a bool?  Silently ignoring and using a default feels "wrong" to me. Maybe that's the convention for reading project properties, but this code is in the PackageSpec reader, and we own the package spec writer, so we can ensure that we never write an invalid value, and therefore we should never expect an invalid value.  Therefore, if this does have an invalid value, it means something is really messed up, so I think an exception should be thrown.